### PR TITLE
Feature/ac 134813 2020 r1 changing current branch2

### PR DESCRIPTION
**Pull request by PetrKorchagin**
An example of switching to another branch and setting the business date is added.

### CODE REVIEW

File: IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json [hunk 1]
```diff
@@ -229,6 +229,115 @@
 						}
 					]
 				},
+				{
+					"name": "New journal transaction with changed business date and branch",
```
Comment (k-s-popova) [inline IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json]: @PetrKorchagin , please change the name to the name of the topic in help.
Do not forget to change the name in the example of response.

File: IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json [hunk 2]
```diff
@@ -229,6 +229,115 @@
...
+								"key": "PX-CbApiBusinessDate",
+								"value": "2018/01/01",
+								"type": "text"
+							},
+							{
+								"key": "PX-CbApiBranch",
+								"value": "SweetLife Store",
+								"type": "text"
+							}
+						],
```
Comment (k-s-popova) [inline IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json:257]: @PetrKorchagin , please add a description of the request. For an example, see other requests in this folder of the collection.

**Review by k-s-popova — CHANGES_REQUESTED** [review]

File: IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json (hunk 1 above)
Comment (PetrKorchagin) [reply IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json]: Done.

File: IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json (hunk 2 above)
Comment (PetrKorchagin) [reply IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json:257]: Done.

File: IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json [hunk 3]
```diff
@@ -229,6 +229,116 @@
...
+							"raw": "{{EndpointURL}}JournalTransaction",
+							"host": [
+								"{{EndpointURL}}JournalTransaction"
+							]
+						},
+						"description": "This example shows how to set the active branch and business date for a request. This process is described in [Change of the Business Date or Current Branch](https://help.acumatica.com/Help?ScreenId=ShowWiki&pageid=9821cff9-4970-4153-a0f8-dbf5758133a7)."
+					},
+					"response": [
+						{
+							"name": "New journal transaction with changed business date and branch",
```
Comment (k-s-popova) [inline IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json]: Please change this name to the name of the example in help (Change of the Business Date or Current Branch)

**Review by k-s-popova — APPROVED** [review]

**Pull request merged**